docs/team/nicollegann.md

```diff
@@ -6,22 +6,63 @@ title: Nicolle's Project Portfolio Page
 ### Project: Harmonia
 
 **Overview**
-Harmonia is a desktop app for students to manage their academic and non-academic tasks and events.
+
+Harmonia is a command-line-based personal task manager, which helps university students to manage their academic tasks. It is written in Java and has a GUI implemented with JavaFX.
 
 Given below are my contributions to the project.
 
 * **Code contributed**: [RepoSense link](https://nus-cs2103-ay2122s2.github.io/tp-dashboard/?search=nicollegann&breakdown=true)
 
-* **Enhancements to implemented**: To be added
 
-* **Contributions to the UG**: To be added
+* **Enhancements implemented**:
+
+  * Evolved AddressBook into Harmonia 
+    * Modified `EditCommand` and `EditCommandParser` by replacing the fields in AB3 (phone, email, address) with fields used in Harmonia (description, deadline) (PR: [#125](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/125)).
```

Comment:
Should the references to the properties of `Task` or `Person` classes be formatted as code?
```suggestion
    * Modified `EditCommand` and `EditCommandParser` by replacing the fields in AB3 (`phone`, `email`, `address`) with fields used in Harmonia (`description`, `deadline`) (PR: [#125](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/125)).
```